VENCore.podspec

```diff
@@ -1,19 +1,14 @@
 Pod::Spec.new do |s|
-  s.name         = "VENCore"
-  s.version      = "3.1.1"
-  s.summary      = "Core Venmo client library"
-  s.description  = <<-DESC
-                   Core iOS client library for the Venmo api
-                   DESC
-
-  s.homepage     = "https://github.com/venmo/VENCore"
+  s.name         = 'VENCore'
+  s.version      = '3.1.2'
+  s.summary      = 'Core Venmo client library'
+  s.description  = 'Core iOS client library for the Venmo api'
+  s.homepage     = 'https://github.com/venmo/VENCore'
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
-  s.author       = { "Venmo" => "ios@venmo.com" }
+  s.author       = { 'Venmo' => 'ios@venmo.com' }
   s.platform     = :ios, '6.0'
-  s.source       = { :git => "https://github.com/venmo/VENCore.git",
-                     :tag => "v#{s.version}"
-  }
+  s.source       = { :git => 'https://github.com/venmo/VENCore.git',
+                     :tag => "v#{s.version}" }
   s.source_files = 'VENCore/**/*.{h,m}'
-  s.dependency 'CMDQueryStringSerialization', '~>0.2.0'
-  s.requires_arc = true
+  s.dependency 'CMDQueryStringSerialization'
```

Comment:
Since this repo is public, and our iOS SDK depends on it, I'm :-1: in general on removing versions from dependencies to keep things simple for peeps who consume our stuff. If we wanna bring simple query parsing into this repo, that's cool with me, except that the iOS SDK is gonna use it on one line too: https://github.com/venmo/venmo-ios-sdk/blob/master/venmo-sdk/Venmo.m#L336

Don't feel too passionate on removing the dependency, especially since we only use it twice in this path. I _do_ feel pretty passionately about destroying dependency version resolution :P